modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -42,6 +43,10 @@ Some features in this release are currently in Technology Preview (TP). These ex
 |====
 |Feature |TP in OCP versions|GA in OCP versions
 
+|Multiple sources for an application
+|4.11, 4.12, 4.13
+|NA
+
```

Comment:
@harrietgrace @iam-veeramalla: I have mentioned this feature in the [OCP RN for GitOps v1.8](https://55921--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html#GitOps-technology-preview_gitops-release-notes.

what OCP versions must I mention in the **TP in OCP versions** column for this feature? Please check the existing data and provide feedback. Thanks!

Reply:
The OCP versions should be all the supported OCP versions for v1.8 GitOps operator.

Reply:
Addressed it.